stories/datagrid.js

```diff
@@ -86,6 +87,8 @@ storiesOf('DataGrids', module)
             <DataGrid.Cell soft>{row.column4}</DataGrid.Cell>
             <DataGrid.Cell align="right" flex="min-width" preventOverflow={false}>
               <IconMenu position="top-right">
+                <MenuItem onClick={() => console.log('onClick: delete row')}>Duplicate row</MenuItem>
```

Comment:
A click on all `MenuItem`s log the same thing, maybe change these to avoid confusion in the future.